openlibrary/plugins/worksearch/code.py

```diff
@@ -114,6 +114,21 @@
 plurals = dict((f + 's', f) for f in ('publisher', 'author'))
 
 
+def depunctuate(s):
```

Comment:
[punctuation is `!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~`](https://docs.python.org/3.9/library/string.html#string.punctuation) in Python so perhaps a more self-documenting function name...
```suggestion
def colonoscopy(s):
```